app/templates/components/dialogs/prioritize-build-modal.hbs

```diff
@@ -0,0 +1,66 @@
+<Modal
+  class="repo-actions-modal"
+  @isVisible={{@isOpen}}
+  @onClose={{fn @onClose}}
+>
+  <div class="repo-actions-modal__top">
+    <h3 class="repo-actions-modal__header">
+      Prioritize your build
+    </h3>
+    <button
+      type="button"
+      title="Close modal"
+      class="pointer repo-actions-modal__close-button"
+      autofocus={{this.focusOnModal}}
+      {{on 'click' (fn @onClose)}}
+    >
+      <SvgImage @name="icon-failed" @class="icon" />
+    </button>
+  </div>
+  <div class="repo-actions-modal__body">
+    <p>
+      Bypass your scheduled build by fast tracking this one.
+    </p>
+    <p class="label-align">
+      SELECT AN OPTION
+    </p>
+    <div>
+      {{#each this.options as |option|}}
+        {{#let (eq this.selectionKey option.key) as |selected|}}
+          <button
+            class="repo-actions-item {{if selected 'repo-actions-item--selected'}}"
+            onclick={{action (mut this.selectionKey) option.key}}
+          >
+            <span class="repo-actions-item__radio"></span>
+            <span class="repo-actions-item__description">
+              {{option.description}}
+            </span>
+          </button>
```

Comment:
Would it be possible to re-use an existing https://github.com/travis-ci/travis-web/blob/master/app/components/forms/form-checkbox.js component here?

Reply:
@artursmirnov: The target here is to implement radio buttons but the link which you have provided is for checkbox.

Reply:
@ShiviStar, you're right, it's checkbox, but it looks and works very much in the same way:

![2020-06-16 at 16 23](https://user-images.githubusercontent.com/4066489/84780168-ec03bf80-afed-11ea-8af8-288c20e8b9b1.gif)

And it's easy to achieve this behavior:

```js
<form.field
  @disableFrame={{true}}
  @value={{eq this.shouldCancel true}}
  @onChange={{action (mut this.shouldCancel) true}}
  as |field|
>
  <field.checkbox>
    Cancel builds
  </field.checkbox>
</form.field>

<form.field
  @disableFrame={{true}}
  @value={{eq this.shouldCancel false}}
  @onChange={{action (mut this.shouldCancel) false}}
  as |field|
>
  <field.checkbox>
    Don't cancel builds
  </field.checkbox>
</form.field>
```
Does it make sense?

Please note that the code above is just an example with fake properties and texts

Reply:
Implemented as suggested.

Reply:
Implemented as suggested.